templates/upstream/upstream_member.erb

```diff
@@ -1 +1,12 @@
-  server     <%= @server %>:<%= @port %>  fail_timeout=<%= @upstream_fail_timeout %><% if @upstream_max_fails -%> max_fails=<%=@upstream_max_fails %><% end %>;
+  server <%= @_server %><% if @params_prepend -%> <%= @params_prepend %><% end -%>
```

Comment:
please convert this into an epp template.

Reply:
I'll convert the template.